apps/src/lib/cli.rs

```diff
@@ -1278,6 +1370,10 @@ pub mod args {
     const OWNER: ArgOpt<WalletAddress> = arg_opt("owner");
     const PROTOCOL_KEY: ArgOpt<WalletPublicKey> = arg_opt("protocol-key");
     const PUBLIC_KEY: Arg<WalletPublicKey> = arg("public-key");
+    const PROPOSAL_OFFLINE: ArgFlag = flag("offline");
+    const PROPOSAL_ID: Arg<u64> = arg("proposal-id");
+    const OPTIONAL_PROPOSAL_ID: ArgOpt<u64> = arg_opt("proposal-id");
```

Comment:
(just to keep them together after sorting)
```suggestion
    const PROPOSAL_ID_OPT: ArgOpt<u64> = arg_opt("proposal-id");
```